src/shared/sam/localLambdaRunner.ts

```diff
@@ -348,6 +348,11 @@ export async function runLambdaFunction(
     config: SamLaunchRequestArgs,
     onAfterBuild: () => Promise<void>
 ): Promise<SamLaunchRequestArgs> {
+    // Verify if Docker is running
+    const dockerResponse = await new ChildProcess(true, 'docker', undefined, 'version').run()
```

Comment:
We use `docker ps` in JB to make sure it's running. Technically `version` does check if it's running (I just checked), but I would call that incidental vs ps that explicitly tries to make a call to the docker daemon.

Reply:
That was my second choice.  I'll make the change.